portfolio/src/main/webapp/script.js

```diff
@@ -25,6 +25,59 @@ function Toggle(idName) {
 		setVisibility(section, idName);
 	}
 }
+const imgFilePath = "/images/album/";
+const images = [ 
+  {
+    "filePath": imgFilePath + "LoveIsABeautifulThing_Vulfpeck.jpg",
+    "link": "https://youtu.be/x0vgMl6n7m0",
+  },
+  {
+    "filePath": imgFilePath + "Mayday_Crush.jpg", 
+    "link": "https://youtu.be/29ycT6fA-Rs",
+  },
+  {
+    "filePath": imgFilePath + "OjitosSonados_Ramona.jpg", 
+    "link": "https://youtu.be/-lJzIUniJOE",
+  },
+  {
+    "filePath": imgFilePath + "Rach2_AnnaFedorova.jpg",
+    "link": "https://youtu.be/rEGOihjqO9w",
+  },
+  {
+    "filePath": imgFilePath + "SeTeOlvida_Ramona.jpg",
+    "link": "https://youtu.be/PIndisFYGB8",
+  },
+  {
+    "filePath": imgFilePath + "TunnelOfLove_haroinfather.jpg",
+    "link": "https://youtu.be/cdlvLZqT3Ok",
+  }];
+var time;
+let index = 0;
```

Comment:
consider doing:

class ImageAnchors {
  constructor() {
     this.index = 0;
     document.getElementById('prev').onclick = () => {
         this.changeImg(true);
     };
     document.getElementById('next').onclick = () => {
         this.changeImg(false);
     };
  }

  changeImg(isPrev) {
     // your code here, but now you can use this.index!
  }
}

Reply:
Thank you! At first when I added only "onclick", it would not let me flip through the images more than once. I came up with a solution using the .addEventListener on clicks.